File: projects/m1/009-vowel-or-consonant/solutions/GiorgioCignitti/VowelorConsonant.py
letter=input("insert a letter: ")

def checkLetter(letter):
 number="1234567890";
 if(letter in number):
   return f"you inserted a number"
 elif (letter=="a" or letter=="e" or letter=="i" or letter=="o" or letter=="u"):
    return f"your letter is a vowel"
 elif(letter=="y"):
    return f"y sometimes y is a vowel, and sometimes y is a consonant"
 else:
    return f"your letter is a consonant"

print (checkLetter(letter))

File: projects/m1/009-vowel-or-consonant/solutions/stefandifede/vowel-or-consonant.js
const userInput = prompt("Insert a letter")

//creating an array of vowels
const vowel = ["a", "e", "i", "o", "u"]

function vowelOrConsonant () {
//if user input is "y" display a different message
    if (userInput == "y") {
        alert("Y is sometimes a vowel and sometimes a consonant")
    }
//if user input is included in vowels array  
    else if (vowel.includes(userInput)) {
        alert(userInput + " is a vowel")
    } else {
        alert (userInput + " is a consonant")
    }
}

vowelOrConsonant()